charts/spire/charts/tornjak-frontend/templates/service.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: {{ include "tornjak-frontend.namespace" . }}
+  name: {{ include "tornjak-frontend.service" . }}
+  labels:
+    {{- include "tornjak-frontend.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  selector:
+    {{- include "tornjak-frontend.selectorLabels" . | nindent 4 }}
+  ports:
+    - name: {{ include "tornjak-frontend.fullname" . }}
+      port: {{ .Values.service.port }}
+      targetPort: 3000
```

```suggestion
      targetPort: http
```

Reference port by name, so we don't have to change port changes in 2 places.